dotnet/src/Connectors/Connectors.Memory.Postgres/README.md

```diff
@@ -88,7 +93,9 @@ BEGIN
 END $$;
 ```
 
-## Migration from older versions
+## Migration the MemoryStore from older versions
```

Comment:
Is it still necessary to keep this section? It is a very early note. Maybe it's time to remove it.